src/index.ts

```diff
@@ -1,5 +1,14 @@
+import fetch from 'node-fetch';
+import FetchInstance from './utils/FetchInstance';
+import DeviceUtils from './utils/DeviceUtils';
+import * as LedgerUtils from './utils/LedgerUtils';
+
+const myFetch = typeof window === 'undefined' ? fetch : window.fetch;
```

Comment:
When would this be true? From what I can tell, in this case `fetch` would always come from node-fetch.